### PR TITLE
Feature/triangle

### PR DESCRIPTION
**Pull request by michaelcapuno**
Add methods:

- hypotenuse (computes the hypotenuse of a triangle)
- sqarea (computes the area of a square)
- triarea (computes the area of a triangle)

### CODE REVIEW

**Review by stevenJhonGil — COMMENTED** [review]
With regards to your second java class CalculateMethod.java, how come you don't have any conflict where in the develop branch has it renamed into CalculateAction.java?

Other than that, everything looks fine (y)

**Comment by stevenJhonGil** [comment]
Also in your documentation, README.md... It lacks this

## Requirements
- JDK 8

Kindly check in the develop branch

File: README.md [hunk 1]
```diff
@@ -15,3 +15,10 @@ $ java Calculate <action-name> <arguments...>
 - `multiply A B` - Multiplies the `A` by `B`
 - `divide A B` - Divides `A` by `B`
 - `mod A B` - Performs modulo operations from integer `A` and `B`
+<<<<<<< HEAD
```
Comment (stevenJhonGil) [inline README.md]: remove these unnecessary lines

**Comment by michaelcapuno** [comment]
fixed merge branch to develop and fix merge conflicts

**Review by stevenJhonGil — APPROVED** [review]
Everything seems to be good. :+1:  :+1:

**Comment by kana-ph** [comment]
@michaelcapuno 

Please replace `rectarea` with `triarea`

This computes the area of the triangle.

Reference: https://en.wikipedia.org/wiki/Triangle#Computing_the_area_of_a_triangle

**Comment by michaelcapuno** [comment]
@kenichigouang @stevenJhonGil  changes has been applied.

**Comment by kana-ph** [comment]
@michaelcapuno 

tag your reviewers, not me

**Review by kenichigouang — APPROVED** [review]
New triarea method is good. :100: :smiley:

**Comment by stevenJhonGil** [comment]
:+1:  Great Job!

**Pull request merged**